src/optionroms.c

```diff
@@ -136,9 +136,19 @@ init_optionrom(struct rom_header *rom, u16 bdf, int isvga)
 
     tpm_option_rom(newrom, rom->size * 512);
 
+    //TODO: Find a way to hide initialisation string of iPXE
+    // It is called everytime and I have no idea how it was hidden before.
+    // For now it can be commented out and seems to work good.
+
+    /*
```

Comment:
please remove commented-out code. Git history should be enough. You can leave comment why it was changed for future reference.